ruby/README.md

```diff
@@ -645,21 +645,25 @@ result = hash.map { |k, v| v + 1 }
 result = hash.map { |_, v| v + 1 }
 ```
 
-### Use '{}' when passing a hash as an argument, but not for keyword args.
+### Do not use a hash literal to destructure keyword args.
```

Comment:
I think the destructuring part is a bit distracting from the main point. How about:
```
### Omit {} when passing options hashes at the end of method calls, to enable them to be converted to keyword args in the future without having to change the calling code.

Reply:
This example doesn't necessarily cover that use case. I can add that as an additional example.